src/NHibernate/Impl/SessionImpl.cs

```diff
@@ -264,16 +264,10 @@ public bool ShouldAutoClose
 			get { return IsAutoCloseSessionEnabled && !IsClosed; }
 		}
 
-		/// <summary>
-		/// Close the session and release all resources
-		/// <remarks>
-		/// Do not call this method inside a transaction scope, use <c>Dispose</c> instead, since
-		/// Close() is not aware of distributed transactions
-		/// </remarks>
-		/// </summary>
+		/// <inheritdoc />
 		public DbConnection Close()
 		{
-			using (BeginContext())
+			using (BeginProcess(true))
```

Comment:
For consistency with Dispose. It does not make it more "aware" of system transactions. The non-awareness of this method is that it closes unconditionally, instead of delaying closing to the transaction completion if called in an ongoing transaction scope with flush from completion enabled.